### PR TITLE
image_rebase_bypass_host_cache: add new case

### PR DESCRIPTION
**Pull request by lolyu**
id:1774452
check if rebase with cache mode `none`, flag `O_DIRECT` is on for open
or openat.

Signed-off-by: lolyu <lolyu@redhat.com>

### CODE REVIEW

**Comment by lolyu** [comment]
@hellohellenmao @YongxueHong Please help to review, thanks.

**Comment by lolyu** [comment]
```
 (1/1) Host_RHEL.m7.u7.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.0.1.x86_64.io-github-autotest-qemu.image_rebase_bypass_host_cache: PASS (1037.57 s)
```

File: qemu/tests/image_rebase_bypass_host_cache.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    remove_snapshots()
+    parse_snapshot_chain(create_snapshot(save_file_to_snapshot()))
+
+    strace_log = os.path.join(test.debugdir, "rebase_bypass.log")
+    top = snapshots[-1]
+    logging.debug("rebase snapshot %s to %s in cache mode 'none'",
+                  top.tag, base.tag)
+    with img_utils.strace(top, trace_events, strace_log):
+        top.base_tag = base.tag
+        fail_on((process.CmdError))(top.rebase)(params, cache_mode="none")
```
Comment (hellohellenmao) [inline qemu/tests/image_rebase_bypass_host_cache.py]: Here is also `source_cache_mode="none"` there.Thanks.

**Comment by lolyu** [comment]
depends on #https://github.com/avocado-framework/avocado-vt/pull/2342

**Comment by lolyu** [comment]
```
 (1/1) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.0.1.x86_64.io-github-autotest-qemu.image_rebase_bypass_host_cache: PASS (334.90 s)
```

**Comment by hellohellenmao** [comment]
(1/1) Host_RHEL.m7.u8.luks.virtio_scsi.up.virtio_net.Guest.RHEL.7.7.x86_64.io-github-autotest-qemu.image_rebase_bypass_host_cache: PASS (1738.91 s)

**Pull request merged**